src/Modules/Block.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Block class.
+ *
+ * This is useful for registering custom gutenberg block to
+ * add `Login with Google` button in desired place.
+ *
+ * Particularly useful in FSE.
+ *
+ * @package RtCamp\GoogleLogin
+ * @since 1.0.0
```

Comment:
Plugin version 1.0.0 was released long time ago, this should be updated to next version we're going to release